### PR TITLE
Add script to update Asana from Bugsag

### PR DESCRIPTION
**Pull request by pbrisbin**
Please see individual commits, script comment, and README details.

### CODE REVIEW

File: debt-evaluation/Main.hs [hunk 1]
```diff
@@ -125,24 +125,19 @@ midPoints points = (xMid, yMid)
...
-quickEnumId :: Integer
-quickEnumId = 1109768955052834
-
-majorEnumId :: Integer
-majorEnumId = 1109768955052835
+toActionabilityField :: Priority -> CustomField
+toActionabilityField priority =
+  CustomEnum actionabilityFieldId "Actionability" actionabilityEnumOptions
+    $ Just enumName
+  where enumName = tshow priority
```
Comment (pbrisbin) [inline debt-evaluation/Main.hs]:
```suggestion
toActionabilityField =
  CustomEnum actionabilityFieldId "Actionability" actionabilityEnumOptions . Just . tshow
```

File: library/Asana/Api/Task.hs [hunk 2]
```diff
@@ -47,22 +48,44 @@ import RIO.Time
...
+  , eoName :: Text
+  }
+  deriving (Eq, Generic, Show)
+
+instance FromJSON EnumOption where
+  parseJSON = genericParseJSON $ aesonPrefix snakeCase
+
+customEnumId :: CustomField -> Maybe Gid
+customEnumId (CustomEnum _ name opts _) =
+  eoGid <$> find ((== name) . eoName) opts
```
Comment (pbrisbin) [inline library/Asana/Api/Task.hs]: This is a bug, it's finding an enum option with the same name as the field name, not the value.

**Review by eborden — APPROVED** [review]
You working towards an `aws` like utility?

**Comment by pbrisbin** [comment]
Good question. I just generally like smaller utilities that can be glued together like this, so it could totally trend that way, but I don't necessary have any real intention here. Just building what I need as I need it, for now.

**Comment by pbrisbin** [comment]
Fixed a typo, decided to have `--quiet` still show warnings (effectively it means don't show `INFO`), and cleaned up history. Will merge on Green.

**Pull request merged**